src/Illuminate/Database/Query/Grammars/SQLiteGrammar.php

```diff
@@ -19,6 +20,32 @@ class SQLiteGrammar extends Grammar
         '&', '|', '<<', '>>',
     ];
 
+    /**
+     * Compiles a single CONCAT value.
+     *
+     * @param array $parts The concatenation parts.
+     * @param string $as The alias to return the entire concatenation as.
+     * @return string
+     */
+    protected function compileConcat(array $parts, string $as)
+    {
+        $compileParts = [];
+
+        foreach ($parts as $part) {
+            if (preg_match('/^[a-z_@#][a-z0-9@$#_]*$/', $part)) {
+                $compileParts[] = $this->wrap($part);
+            } else {
+                // Removes quotes from quoted strings
+                if (preg_match('/^[\'"]*(.*?)[\'"]*$/', $part, $matches)) {
+                    $part = $matches[1];
+                }
```

Comment:
Could be simplified to just `$part = trim($part, '\'"');`

Reply:
@multiwebinc Good suggestion :)